openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -138,7 +138,8 @@ def test_set_get_credit_requirements(self):
                 "display_name": "Grade",
                 "criteria": {
                     "min_grade": 0.8
-                }
+                },
+                "order": 1,
```

Comment:
Nitpick: Since now Grade requirement will always have `"order": 0,` (first one in credit requirements) so please make changes in all examples accordingly.